effector_mass: 0.0
rotation:      [0.0, 0.0, 0.0, 1.0]
force_offset:  [0.0, 0.0, 0.0]
torque_offset: [0.0, 0.0, 0.0]
mass_center:   [0.0, 0.0, 0.0]
